### ISSUE TITLE
docker image not working

### ISSUE THREAD

**Issue by rnd-debug** [totaljs/flow#68]
In order to reproduce, follow the steps given in the documentation :( 
```
docker pull totalplatform/flow
docker run -p 8000:8000 totalplatform/flow
```
=> Error : " standard_init_linux.go:219: exec user process caused: exec format error "
No other Docker containers running in parallel. Am I missing a requirement ?

**Comment by molda** [comment]
Hi @rnd-debug i think this has to do with the os you run it on. Are you by any chance on MacOS with M1 cpu?

**Comment by rnd-debug** [comment]
It is a standard Debian 9 VM. 
```
>> uname -a
Linux debianRGA 4.9.0-9-amd64 #1 SMP Debian 4.9.168-1+deb9u5 (2019-08-11) x86_64 GNU/Linux
```
![image](https://user-images.githubusercontent.com/58293036/160756214-7e7baaab-0d33-43ea-a575-b71881f6dc75.png)

**Comment by molda** [comment]
So it seem that the issue is on our end with the way we built the docker image. I'll let you know when it's fixed.

**Comment by molda** [comment]
Ok i have updated the image. Now it supports both amd64 as well as arm64.
It was previously build on MacOS M1 so it only supported arm64.

Unfortunately i don't have a way to test it right now, but will do later this week.

**Comment by molda** [comment]
I managed to test it on Ubuntu server. Seems to work ok.
`Linux vm29663 5.4.0-90-generic #101-Ubuntu SMP Fri Oct 15 20:00:55 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux`

**Comment by rnd-debug** [comment]
Yep, confirming, looks better on my side as well: able to start the container.
Thank you!
